docs/roadmap.md

```diff
@@ -1,7 +1,4 @@
 ## Upcoming features
```

Comment:
I think we should remove todo and roadmap from the project, and instead use GitHub issues

Reply:
Agreed!